jsonnet/config.libsonnet

```diff
@@ -269,6 +271,22 @@
           enabled: false,
         },
       },
+      zfsLocalPV: {
+        // To generate manifests for zfs localpv. If set, manifests will be generated.
+        enabled: true,
+        // To generate dashboards configMap yamls. If set, dashboards will be appended in grafana-dashboardDefinition yaml.
+        dashboards: true,
+        // To generate prometheusRule yamls. If set, prometheusRule will be generated.
+        alertRules: false,
+        // ServiceMonitor configuration
+        serviceMonitor: {
```

Comment:
Should this be true or false?

Reply:
There are no alert rules, serviceMonitor and podMonitor for zfs localpv. Therefore it is set as false.
Dashboard is built using node-exporter metrics.